framework/src/meshgenerators/AdvancedExtruderGenerator.C

```diff
@@ -993,6 +1072,10 @@ AdvancedExtruderGenerator::generate()
                                                             input_nodeset_map.end());
 
   mesh->set_isnt_prepared();
+  // Creating the layered meshes creates a lot of leftover nodes, notably in the boundary_info,
+  // which will crash both paraview and trigger exodiff. Best to be safe.
+  if (extruding_quad_eights)
+    mesh->prepare_for_use();
```

Comment:
If the only thing we need here is orphaned node removal, can we get away with `mesh->remove_orphaned_nodes()`?  Or maybe with `mesh->renumber_nodes_and_elements()`, if contiguous numbering is preferred and renumbering is allowed?